assembly/Hash.ts

```diff
@@ -64,8 +64,8 @@ export class Hash {
     /**
     * @description Instantiates Hash from u8[] SCALE encoded bytes
     */
-    static fromU8a (input: u8[]): Hash {
-        return new Hash(input);
+    static fromU8a (input: u8[], index: i32 = 0): Hash {
```

Comment:
Maybe we should add inline docs with `@params` tag so that users will know what input and more importantly `index` stands for (and ofcourse understand that its optional one)